modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -285,8 +285,23 @@ function getValidFieldsTypes($module, $field){
     return $valid_type;
 }
 
+/**
+ * This function retrieves a module's fields.
+ *
+ * @param string $module
+ * @param string $fieldname
+ * @param string $aow_field
+ * @param string $view name
+ * @param string $value
+ * @param string $alt_type
+ * @param string $currency_id
+ * @param array $params
+ * @param bool $reports_csv determines if returned field is for building a csv
+ * @return string module fields
+ */
 
-function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value = '', $alt_type = '', $currency_id = '', $params= array()){
+function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value = '', $alt_type = '', $currency_id = '', $params= array(), $reports_csv)
```

Comment:
$reports_csv needs to = false to prevent PHP error "Missing argument 9 for getModuleField(), called in AOR_Report.php on line 753".